tests/functional/aws-node-sdk/test/bucket/put.js

```diff
@@ -224,5 +224,23 @@ describe('PUT Bucket - AWS.S3.createBucket', () => {
                 });
             });
         });
+
+        describe('bucket creation with ingestion location', () => {
+            after(done =>
+                bucketUtil.s3.deleteBucket({ Bucket: bucketName }, done));
+            it('should create bucket with location and ingestion', done => {
+                bucketUtil.s3.createBucketAsync(
+                    {
+                        Bucket: bucketName,
+                        CreateBucketConfiguration: {
+                            LocationConstraint: 'us-east-1:ingest',
+                        },
+                    }, (err, res) => {
+                    assert.ifError(err);
+                    assert.strictEqual(res.Location, `/${bucketName}`);
+                    return done();
```

Comment:
An assertion that the ingest property is set and that the location constraint of the bucket is well formed would be good.